### DISCUSSION TITLE
ilogtail don't register config_server??

### DISCUSSION THREAD

**Discussion by wentao1994** [alibaba/ilogtail#608] [Help] [answered]
system info :  5.15.0-53-generic #59-Ubuntu SMP Mon Oct 17 18:53:30 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
ilogtail version: ilogtail-1.3.0
config:
<img width="346" alt="image" src="https://user-images.githubusercontent.com/19541731/212832567-860ec18e-c68a-492a-aba4-e44605bb4458.png">

request
<img width="480" alt="image" src="https://user-images.githubusercontent.com/19541731/212832616-36b36579-1716-43a0-98ac-a1e7b4d88de9.png">

code:
![image](https://user-images.githubusercontent.com/19541731/212832724-369fef8b-8e57-4982-bcd1-b6d070b618ae.png)

why is the list empty?

**Comment by wentao1994** [comment]
May be , I need to recompile it 🤔

**Comment by messixukejia** [comment, marked as answer]
The pr is not merged completed. https://github.com/alibaba/ilogtail/pull/506/files

We will accelerate progress recently.
